### PR TITLE
Add customizable worker loglevel

### PR DESCRIPTION
**Pull request by fmhoeger**
#### Added 
- Added new config variable `loglevel_worker` for worker processes

### CODE REVIEW

**Comment by pmcharrison** [comment]
To test - go to a PsyNet demo that uses background processes, for example the gibbs demo, and put some logger commands at different logging levels into `async_post_grow_network`. Check which ones get printed.

```py
logger.info("Hello")
logger.warn("Agh")
```

File: dallinger_scripts/worker.py [hunk 1]
```diff
@@ -25,10 +25,15 @@ def main():
...
+    config = get_config()
+    if not config.ready:
+        config.load()
+
+    loglevels = ["DEBUG", "INFO", "WARN", "ERROR", "CRITICAL"]
+    loglevel = os.environ.get("loglevel", loglevels[config.get("loglevel_worker")])
     logging.basicConfig(
         format="%(asctime)s %(message)s",
-        level=getattr(logging, log_level, logging.WARN),
+        level=getattr(logging, loglevel, logging.WARN),
```
Comment (pmcharrison) [inline dallinger_scripts/worker.py:36]: Isn't this the line you need to change, specifically the `logging.WARN`?

Comment (pmcharrison) [reply dallinger_scripts/worker.py:36]: Need to debug this using the remote debugger, e.g. rpdb

**Comment by fmhoeger** [comment]
@pmcharrison I think you can have a look again now.

As I used a PsyNet to test, the problem was really in PsyNet where the loglevel is hardcoded in `psynet/utils.py`: 
```
def get_logger():
    logging.basicConfig(level=logging.INFO)
    return logging.getLogger()
```
Once I commented that line I could set the log level from `config.txt`. The default is `logging.WARNING`, i.e. when `loglevel_worker` is not set explicitly.

**Comment by pmcharrison** [comment]
> @pmcharrison I think you can have a look again now.
> 
> As I used a PsyNet to test, the problem was really in PsyNet where the loglevel is hardcoded in `psynet/utils.py`:
> 
> ```python
> def get_logger():
>     logging.basicConfig(level=logging.INFO)
>     return logging.getLogger()
> ```
> 
> Once I commented that line I could set the log level from `config.txt`. The default is `logging.WARNING`, i.e. when `loglevel_worker` is not set explicitly.

!! Well spotted!!!

**Review by pmcharrison — APPROVED** [review]

**Pull request merged**